.github/workflows/python-ci-wheel.yml

```diff
@@ -142,8 +142,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04]
-        python-version: [3.9]
+        os: [ubuntu-20.04]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Actually this makes no sense to me. Why do you want different source distributions for each python version? It broke the action. So you should either remove it or fix the file name conflict it creates.

Reply:
ok, for which version then? I would guess 3.6, but you maybe prefer the latest?

Reply:
3.9 used to work, so I don't see a reason to change this. Why do you want 3.6? In other words: what problem have you encountered with the source distribution?

Reply:
Beside the question of having sdists for multiple python versions, there seems to be a real issue with the naming of the artifacts: cf. #2069 (which wasn't introduced but revealed by this PR here)

Reply:
I set it back https://github.com/rism-ch/verovio/commit/633ee29af3fd1b1cff8759fe2bd1a78abad16c85
If there is a problem with the sdist open a dedicated issue.